.agents/journal/scribe-journal.md

```
@@ -70,9 +70,32 @@
 
 ---
 
+## 2026-05-22 - CLI Reference Audit & Update - COMPLETE
+
+**Verification:**
+- Audited `clients/agent-runtime/src/main.rs` and `lib.rs` to identify undocumented CLI features.
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)